### PR TITLE
Project Proposal Ideas/Contribution Ideas Added

### PR DESCRIPTION
**Pull request by DavidCendejas**
Added my brainstorming ideas for my own project and the projects I might contribute to.

### CODE REVIEW

**Review by catw101 — DISMISSED** [review]
Restore `tkp2108.md`

File: tkp2108.md [hunk 1]
```diff
@@ -4,11 +4,13 @@ Name: Tim Paine
 
 UNI: tkp2108
 
-Github: [link](https://github.com/timkpaine)
+GitHub: [link](https://github.com/timkpaine)
+
+GitHub README: [link](https://github.com/timkpaine/timkpaine/blob/main/README.md)
```
Comment (timkpaine) [inline tkp2108.md:9]: unnecessary changes in this PR

**Review by timkpaine — CHANGES_REQUESTED** [review]
